### PR TITLE
Hello-back-end

### PR DESCRIPTION
**Pull request by hasnatali1947**
Features
API database
API rendering
Access to all apps
Random generate greetings

front-end Url = https://github.com/hasnatali1947/Hello_frontEnd.git

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @hasnatali1947, 

# Invalid Code Review Request

descarga

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the link in the README file, please 🙏🏽 add the link to the front end to the README of the back end, and also add the link to the back end in the README of the front end.

> In the README file for the Rails API, add a link to your React app.
> Also, in the README file for the React app, add a link to your Rails API.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/api/v1/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def index
+        @messages = Message.all
```
Comment (brainconnect93) [inline app/controllers/api/v1/greetings_controller.rb:5]: - [ ] Great effort working on your project, but currently, On the front end I realize the random greeting is not rendered on the screen. I would kindly suggest you work on it and be sure the greetings are showing on the screen
Below is a video demonstration

Video 👇 

[screen-capture (1).webm](https://github.com/hasnatali1947/Hello_backEnd/assets/78634841/a44a57a5-b8fe-40f0-b062-98ecf8b8e647)


>N.B: I highlighted the code above just to use it as an inline comment, that is not where the problem is coming from. The problem is on the front-end page, and the place where the error is coming from has been stated in the video

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @hasnatali1947, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- "Hello world" project with `react-rails` gem ✔️
- Implement the `API` in the backend correctly ✔️
- Add link to the `Frontend` in the README file ✔️ 
- Good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/api/v1/greetings_controller.rb (hunk 1 above)
Comment (rloterh) [reply app/controllers/api/v1/greetings_controller.rb:5]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Hasnat,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Backen API correctly connects with Reat frontend!
🎲 Random greeting are generating correctly!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Let's ensure to include the links to frontend in the ReadMe of the backend, and the vice versa
- Let's improve the look of the ReadMe by removing the underlined:

image

Implementing the above change(s) will make your ReadMe stand out even better! 😉

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**